poc/pubsub/cp

```diff
@@ -0,0 +1,107 @@
+export type Json =
```

Comment:
Hm. Did you pipe the output of `supabase gen types typescript --project-id <asdasdsad>` to the wrong file